packages/cli/src/index.ts

```diff
@@ -149,6 +149,10 @@ yargs
           array: true,
           type: 'string',
         },
+        'skip-config-lint': {
```

Comment:
I wouldn't introduce this flag. I think it's better to run lint config with `warn` severity in order not to break anything for customers.

Reply:
This flag just skips lint of the config and in case you dont like your CLI output to be polluted with config lint result - you can skip it

Reply:
Not sure I understand how it could break anything, because config lint doesnt interrupt definition lint in case of errors

Reply:
Ah, I thought you introduced this option because config linting may throw unwanted errors. For this cases, it may sense to use this flag.